test/cases/compile_errors/invalid_struct_field.zig

```diff
@@ -12,8 +12,10 @@ export fn g() void {
 }
 
 // error
-// backend=stage1
+// backend=stage2
 // target=native
 //
-// tmp.zig:4:6: error: no member named 'foo' in struct 'A'
-// tmp.zig:10:16: error: no member named 'bar' in struct 'A'
+// :4:7: error: no field named 'foo' in struct 'tmp.A'
+// :1:11: note: struct declared here
+// :10:17: error: no field named 'bar' in struct 'tmp.A'
+// :1:11: note: struct declared here
```

Comment:
@kubkon can you take a look at this test? I tried fixing it like in d1e46006758a82f039d23ea26030e3b685a5fa86 but that broke other tests.

Reply:
Will do!